collections/submissions.js

```diff
@@ -38,3 +38,25 @@ Submissions.inTableFormat = function(promptsInOrder) {
     return responsesInOrder;
   });
 }
+
+Submissions.exportCsvFormattedString = function() {
```

Comment:
we should eventually use a robust csv library to do this which will handle things like commas in questions

I'll add a task to trello for it - this is great for our demo!